### PR TITLE
Update kustomize base when setting image tags

### PR DESCRIPTION
**Pull request by jessesuen**
* The kustomize base will be updated with new image tags when `make manifests` is invoked
* Also fixes the sed command which broke on Mac OS X

### CODE REVIEW

File: hack/update-manifests.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (twz123) [inline hack/update-manifests.sh]: Interesting. Why the change to bash? The script looks pretty much POSIXish.

Comment (jessesuen) [reply hack/update-manifests.sh]: Actually it probably doesn't need to change. I think I was experimenting and never changed it back. Will update.

**Review by alexmt — APPROVED** [review]
